include/envoy/access_log/access_log.h

```diff
@@ -71,11 +71,20 @@ typedef std::shared_ptr<Instance> InstanceSharedPtr;
 
 /**
  * Interface for access log formatter.
+ * Formatters combine the output of FormatterProviders into a log output line.
```

Comment:
Looking at this interface, it's a bit unclear how they do this combining; they don't take FormatterProviders as arguments to format..

Reply:
Agreed. The combining takes place in the subclass which isn't entirely clear at this level. I changed to `Formatters provide a complete access log output line for the given headers/trailers/stream.`